MMR.Randomizer/Resources/REQ_CASUAL.txt

```diff
@@ -46587,12 +46818,7 @@
           "Use Ice Arrow"
         ],
         [
-          "FD Difficult Jumps",
-          "Use Ice Arrow"
-        ],
-        [
-          "FD Difficult Jumps",
-          "ItemHookshot"
+          "FD Difficult Jumps"
```

Comment:
How do you get to the barrels with just FD? Is it implicit via Access Gyorg + Kill Gyorg?

Reply:
FD can just jump straight to the barrels. It surprised me when I first tried it!